vars/common.yml

```diff
@@ -1,46 +1,52 @@
 ---
+maintenance_mode: live
```

Comment:
Could you explain this line for me please? It reads like maintenance mode would be on by default and I couldn't find anything else in our aws repo in terms of setting this variable so it seems new

Reply:
That's the existing nginx `mode` flag from Terraform. When this is set to `maintenance_mode: maintenance` nginx will be deployed with a maintenance page.

The environment variable is just called `DM_MODE`, but I felt like it's worth expanding the name a bit here, even though I agree it's not ideal.

Reply:
Cheers. Yeah it's not the key that is the issue so much, more the value that can be a little confusing.